sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/context/AzureGlobalConfigurationEnvironmentPostProcessor.java

```diff
@@ -91,11 +105,30 @@ public int getOrder() {
             this.springPropertyName = "spring.cloud.azure." + springPropertyName;
             this.converter = converter;
         }
+
+        private static Function<String, String> convertMillisToDuration() {
```

Comment:
I think here can just use duration, no need to convert it to string again?

Reply:
done